tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "allowJs": false,
     "noImplicitAny": true,
     "moduleResolution": "node",
+    "resolveJsonModule": true,
```

Comment:
With this option, our `package.json` file will be present in the `build` folder along with the compiled code.
This will break the main path of the package. So please update the main `package.json` path to `build/src/index.js`.
Also add `"rootDir": "."` [option](https://www.typescriptlang.org/tsconfig#rootDirs) here to make the compilation process clearer.